docs/adr/0005-replace-apache-with-nginx-gunicorn.md

```diff
@@ -0,0 +1,130 @@
+# ADR 5: Replace Apache with Nginx + Gunicorn for Manager Service
+
+- **Author(s)**: [Mikolaj Kasprzak](https://github.com/MikolajKasprzak)
+- **Date**: 2025-10-21
+- **Status**: `Accepted`
```

Comment:
The primary concern is that the Manager service runs as root, which can be fixed **without major architectural changes**. We should first clean up technical debt - simplify/remove init scripts, fix permissions, and remove legacy dependencies - similar to what was done for the Controller in [PR #235](https://github.com/open-edge-platform/scenescape/pull/235).

Long-term separation of the HTTP server and Manager service makes sense, so this ADR should focus solely on that aspect.